js/main.js

```diff
@@ -229,11 +229,16 @@ $(window).bind('keydown', function(e) {
         // Select all text in address bar
         $('.js-navAddressBar').select();
         break;
+      case config.keyShortcuts.save:
+        // Trigger any visible .saveForm button
+        // .last() will make sure you'll only click one button if 2 are visible (listing form)
+        $('.js-saveForm:visible').last().trigger('click');
```

Comment:
It will be a kind of redundant code (for each view), you need a mechanism to tell what event to trigger from the js/main.js etc. The current code only requires you to add a .saveForm class to the button you want to allow the key shortcut for.